packages/base/style/leftPanel.css

```diff
@@ -108,6 +108,12 @@
   padding-right: 4px;
 }
 
+.jp-gis-layerIcon.jp-mod-hidden,
+.jp-gis-sourceIcon.jp-mod-hidden {
+  stroke: red;
```

Comment:
We should not use hard-coded CSS colors, as it will not work well with the theme system.

Also icons colors are set broadly across JupyterLab to provide a coherent UI.

May I suggest to reuse the icons from JupyterCAD instead? The [visibilityOff](https://github.com/jupytercad/JupyterCAD/blob/main/packages/base/style/icon/visibilityOff.svg) and [visibility](https://github.com/jupytercad/JupyterCAD/blob/main/packages/base/style/icon/visibility.svg) are a bit easier to read at the same page-zoom level:

## JupyterGIS:
![Screenshot From 2024-12-13 09-16-05](https://github.com/user-attachments/assets/59eb2bf9-d295-4889-8bb7-79ae1fd60af4)

## VS. JupyterCAD:
![Screenshot From 2024-12-13 09-16-23](https://github.com/user-attachments/assets/99203ef7-7485-47b5-914e-031f6ec91aef)

Reply:
Looks good!